src/components/Gallery/Gallery.tsx

```diff
@@ -103,7 +106,7 @@ export const GalleryMedia = ({
           />
         </Button>
       </div>
-      {isLead && <div className="break" />}
```

Comment:
Just checking - is this change for style consistency, or a technical reason? Just wondering!

Reply:
It is (unfortunately) purely for layout/styling reasons. Loathe as I am to use non-semantic styles and coupled CSS+HTML, @chriddell and I talked thru options at the time and realised that this was really the only way to make row breaks work

Reply:
Fair enough - there's always cases where the non-idealistic solution is the best option...